scene/2d/canvas_item.cpp

```diff
@@ -61,14 +63,17 @@ void CanvasItemMaterial::init_shaders() {
 
 void CanvasItemMaterial::finish_shaders() {
 
+	memdelete(dirty_materials);
+	dirty_materials = NULL;
```

Comment:
Isn't that redundant with `memdelete`?

Reply:
(Unless I've really missed something reading it), memdelete doesn't null the pointer